client/src/components/Layout/Footer.css

```diff
@@ -0,0 +1,18 @@
+.iconContainer {
```

Comment:
Added it to src/css . i did not add it to public because this is not possible in react applications. The public folder should only be used for icons/images that may be publicly accessed